lib/support/localized_block.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Jekyll
+  class LocalizedBlock < Liquid::Block
```

Comment:
If this file is refactored to be the `Base` module, we should move and rename it to `lib/jekyll-plugin-platoniq-journal/base.rb` changing only:

```suggestion
module JekyllPluginPlatoniqJournal
  module Base
```